scripts/core/editor3/reducers/find-replace.tsx

```diff
@@ -174,22 +186,27 @@ const render = (state) => {
     const {pattern, index, caseSensitive} = state.searchTerm;
     const es = state.editorState;
 
+    let changedContent = false;
+
     let {content, editorState} = clearHighlights(es.getCurrentContent(), es);
+    let newContent = content;
 
     if (!pattern) {
         return {...state, editorState};
     }
 
-    const changedContent = forEachMatch(content, pattern, caseSensitive, (i, selection) => {
-        content = Modifier.applyInlineStyle(
-            content,
+    newContent = forEachMatch(content, pattern, caseSensitive, (i, selection, block, _newContent) => {
```

Comment:
I'd put here `const newContent` and remove the above `let newContent = content` as that value is not being used at all